static/js/zamboni/devhub-api.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function () {
+  var KEY_TIMEOUT = 30000;
+
+  function clearKey() {
+    $('#jwtsecret').val('').hide();
+    $('#show-key').show();
+  }
+
+  function regenKey() {
```

Comment:
can you update the names of these functions after refactoring? `regenKey` doesn't sound relevant.